src/navigations/Flow3Navigator/index.js

```diff
@@ -28,44 +42,272 @@ const Flow3Navigator = () => {
+            setImgUri={setImgUri}
+            setDuration={setDuration}
+            time={time}
+            setTime={setTime}
+          />
+        )}
+      />
+      <Stack.Screen
+        name="Preview1"
+        options={({ navigation }) => ({
+          header: () => (
+            <BenefitProgressHeader
+              current={2}
+              total={5}
+              navigation={navigation}
+            />
+          )
+        })}
+        children={(props) => (
+          <PreviewScreen
+            {...props}
+            prompt={prompts[0].prompt}
+            imgUri={imgUri}
+            duration={duration}
+            time={time}
+            nextScreen={'Breathe'}
+          />
+        )}
+      />
+      <Stack.Screen
+        name="Preview2"
+        options={({ navigation }) => ({
+          header: () => (
+            <BenefitProgressHeader
+              current={4}
+              total={5}
+              navigation={navigation}
+            />
+          )
+        })}
+        children={(props) => (
+          <PreviewScreen
+            {...props}
+            prompt={prompts[1].prompt}
+            imgUri={imgUri}
+            duration={duration}
+            time={time}
+            nextScreen={'ReviewBenefitList'}
+          />
+        )}
+      />
+      <Stack.Screen
+        name="PromptText1"
+        options={({ navigation }) => ({
+          header: () => (
+            <BenefitProgressHeader
+              current={2}
+              total={5}
+              navigation={navigation}
+            />
+          )
+        })}
+        children={(props) => (
+          <PromptText
+            {...props}
+            prompt={prompts[0].prompt}
+            setResponse={setShortTermBenefits}
+            nextScreen={'Breathe'}
+            audioScreen={'PromptAudio1'}
+            videoScreen={'PromptVideo1'}
+          />
+        )}
+      />
+      <Stack.Screen
+        name="PromptText2"
+        options={({ navigation }) => ({
+          header: () => (
+            <BenefitProgressHeader
+              current={4}
+              total={5}
+              navigation={navigation}
+            />
+          )
+        })}
+        children={(props) => (
+          <PromptText
+            {...props}
+            prompt={prompts[1].prompt}
+            setResponse={setLongTermBenefits}
+            nextScreen={'ReviewBenefitList'}
+            audioScreen={'PromptAudio2'}
+            videoScreen={'PromptVideo2'}
           />
         )}
       />
-      <Stack.Screen name="PromptText" component={PromptText} />
+      <Stack.Screen
+        name="Breathe"
+        options={({ navigation }) => ({
+          header: () => (
+            <BenefitProgressHeader
+              current={3}
+              total={5}
+              navigation={navigation}
+            />
+          )
+        })}
+        children={(props) => (
+          <BreatheScreen
+            {...props}
+            prompt={prompts[1]}

```

Comment:
prompt from successful habits, though not harmful due to default in `BreatheScreen`